pkgs/development/libraries/libgudev/default.nix

```
@@ -23,11 +25,11 @@ stdenv.mkDerivation rec {
 
   nativeBuildInputs = [
     pkg-config
-    gobject-introspection
     meson
     ninja
-    vala
-  ];
+    glib
+  ] ++ lib.optional withIntrospection gobject-introspection
+    ++ lib.optional withVala vala;
```

This file already switched to the clean style so could you please preserve it?

```suggestion
  ] ++ lib.optionals withIntrospection [
    gobject-introspection
  ] ++ lib.optionals withVala [
    vala
  ];
```

---

I didn't know this was considered the 'clean style', but thanks for pointing out that that's preferred. I like it better too.